src/states/Preroll.js

```diff
@@ -26,8 +26,13 @@ export default class Preroll extends AdState {
       player.trigger('adtimeout');
     }, timeout);
 
+    if (!this.inAdBreak() && !this.isContentResuming()) {
+      player.ads._shouldBlockPlay = true;
```

Comment:
In init you would never be in ad break or content resuming, so this check isn't needed.

Reply:
This is mostly defensive since the flag is used for both cancelContentPlay and playMiddleware and the timing of when it is set is important.

Reply:
I would also like to avoid retesting it currently